src/main/resources/reference.conf

```diff
@@ -242,6 +255,19 @@ cassandra-snapshot-store {
   # Port of contact points in the cluster.
   # Will be ignored if the contact point list is defined by host:port pairs.
   port = 9042
+  
+  # An implementation of akka.persistence.cassandra.ContactPointsProvider
+  # can be used to lookup the contact points of the Cassandra cluster
+  # asynchronously instead of given them in the configuration. If this
+  # is specified the contact-points and port properties will be ignored.
+  # Define fully qualified class name of the implementation of the 
+  # akka.persistence.cassandra.ContactPointsProvider. It may optionally
+  # have a constructor with a ActorSystem parameter.
+  contact-points-provider = ""
+  
+  # The identifier that will be passed as parameter to the
+  # ContactPointsProvider.lookup method. 
+  contact-points-provider-id = ""
```

Comment:
I see problems with the lifecycle dependency of such actor. How would you make sure that it is started before this persistence extension needs it? And you would not be able to stop such an actor, since you would not know when it is is not needed any more.

Do you see any problems implementing this simple interface?